app/code/Magento/Config/Console/Command/ConfigShowCommand.php

```diff
@@ -150,11 +150,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $configPath = $this->pathResolver->resolve($this->inputPath, $this->scope, $this->scopeCode);
             $configValue = $this->configSource->get($configPath);
 
-            if (empty($configValue)) {
-                $output->writeln(sprintf(
-                    '<error>%s</error>',
-                    __('Configuration for path: "%1" doesn\'t exist', $this->inputPath)->render()
-                ));
+            if ($configValue == null && empty($configValue)) {
```

Comment:
`empty` php function is returning true for both `null` and `0` arguments, so I'd check if `$configValue` is an empty array considering `RuntimeConfigSource::get` implementation
```suggestion
            if (is_array($configValue) && empty($configValue)) {
```